src/rqt_plot/rosplot.py

```diff
@@ -101,12 +103,56 @@ def __init__(self, node, topic, start_time):
         topic_type, real_topic, fields = get_topic_type(node, topic)
         if topic_type is not None:
             self.field_evals = generate_field_evals(fields)
-            data_class = get_message(topic_type)
+            data_class = get_message_class(topic_type)
```

Comment:
What's the reason for this change? IMO, it seems like a backwards step. I would prefer to use the utility further upstream, all else being equal.